examples/specs.py

```diff
@@ -1,71 +1,68 @@
+import setup  # noqa
 import maple as mp
 
 
 def main():
     stream_id = "24fa0ed1c3"
     mp.stream(stream_id)
+    mp.set_logging(True)
     mp.run(spec_a, spec_b, spec_c, spec_d, spec_e, spec_f, spec_g)
```

Comment:
Improvement: generate report can be called through mp run as an optional argument